<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayoutWin2</class>
 <widget class="QMainWindow" name="LayoutWin2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>461</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_income">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;收益</string>
         </property>
         <property name="buddy">
          <cstring>spin_income_min</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_drawdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>3</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;最大回撤</string>
         </property>
         <property name="buddy">
          <cstring>spin_drawdown_min</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_sharp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Sharp比</string>
         </property>
         <property name="buddy">
          <cstring>spin_sharp_min</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="labelAlignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="formAlignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_min">
         <property name="text">
          <string>最小值</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_max">
         <property name="text">
          <string>最大值</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="spin_income_min"/>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="spin_income_max"/>
       </item>
       <item row="2" column="0">
        <widget class="QDoubleSpinBox" name="spin_drawdown_min"/>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="spin_drawdown_max"/>
       </item>
       <item row="3" column="0">
        <widget class="QDoubleSpinBox" name="spin_sharp_min"/>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="spin_sharp_max"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_start">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="text">
        <string>開始</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
